analytics/models/java.py

```diff
@@ -0,0 +1,25 @@
+from analytics.models.models import SubmissionAnalytics
+from django.db import models
+from course.fields import JSONField
+
+
+class JavaSubmissionAnalytics(SubmissionAnalytics):
+    lines = models.IntegerField(default=0)
+    blank_lines = models.IntegerField(default=0)
+    comment_lines = models.IntegerField(default=0)
+    import_lines = models.IntegerField(default=0)
+    cc = models.IntegerField(default=0)
+    method = models.IntegerField(default=0)
+    operator = models.IntegerField(default=0)
+    operand = models.IntegerField(default=0)
+    unique_operator = models.IntegerField(default=0)
+    unique_operand = models.IntegerField(default=0)
+    vocab = models.IntegerField(default=0)
```

Comment:
better name. Most of these names are only meaningful to you. Try to name them in a way that anyone can understand what they are. If some cases if you can't come up with a name at least provide a description.